iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonParser.m

```diff
@@ -16,6 +16,8 @@ @implementation MSMapGeoJsonParser {
   MSMapElementLayer *mLayer;
 };
 
```

Comment:
This declaration doesn't look like it's in the right spot. Put with other member declarations a few lines up.

Reply:
Placing it inside the brackets would make this variable an instance variable. Right now it is static and will be shared among all instances of MSMapGeoJsonParser. If your intent was to make it global please rename it to sDidWarn.

Reply:
Moved inside to make it an instance variable.